src/components/chat-view/Chat.tsx

```diff
@@ -75,6 +75,8 @@ const Chat = forwardRef<ChatRef, ChatProps>((props, ref) => {
   const { settings } = useSettings()
   const { ragEngine } = useRAG()
 
+  const chatContainerRef = useRef<HTMLDivElement>(null)
```

Comment:
Where is this used?

Reply:
I forgot to erase this. It was used for dialog container before.